Content.Shared/PlantAnalyzer/PlantAnalyzerScannedSeedPlantInformation.cs

```diff
@@ -0,0 +1,104 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.PlantAnalyzer;
+
+/// <summary>
+///     The information about the last scanned plant/seed is stored here
+/// </summary>
+[Serializable, NetSerializable]
+public sealed class PlantAnalyzerScannedSeedPlantInformation : BoundUserInterfaceMessage
+{
+    public readonly NetEntity? TargetEntity;
+
+    public string SeedName;
+    public string Endurance;
+    public string SeedYield;
+    public string Lifespan;
+    public string Maturation;
+    public string GrowthStages;
+    public string SeedPotency;
+    public string Repeat;
+    public string SeedChem;
+    public string SeedMutations;
+    public string PlantSpeciation;
+    public string ExudeGases;
+    public Boolean IsTray;
+    public Boolean ScanMode;
+
+    public string NutrientConsumption;
+    public string WaterConsumption;
+    public string IdealHeat;
+    public string HeatTolerance;
+    public string IdealLight;
+    public string LightTolerance;
+    public string ToxinsTolerance;
+    public string LowPresssureTolerance;
+    public string HighPressureTolerance;
+    public string PestTolerance;
+    public string WeedTolerance;
```

Comment:
Use appropriate types for these and only turn them to strings in client UI code.